ExerciseTracker/Tests/UITests/ExerciseTrackerUITestsLaunchTests.swift

```diff
@@ -0,0 +1,25 @@
+import XCTest
```

Comment:
Just wondering if this file is needed, it looks like just an import and a commented out test (but I could be missing some swift magic!)

Reply:
You're spot on, that one isn't needed in my case (from my reading this file is used more for snapshot testing). I think it should have been deleted before the big merge happened. The commenting out was me initially testing if it broke anything by not being there as I hadn't seen it before 😅